src/components/elements/Tabs/Default/TabButton.tsx

```diff
@@ -34,21 +34,22 @@ export const TabButton = ({
       className={classNames(
         className,
         `w-full items-center focus:outline-none disabled:text-neutral-900`,
-        "border",
+        "",
         item.done && "peer",
-        lastItem || selected ? "border-b" : "border-b-0",
+        lastItem || selected ? "" : "",
         selected
-          ? "border-neutral-100 border-r-white bg-white text-neutral-1000 peer-aria-checked:shadow-t-secondary"
+          ? `border-l-4 border-l-[#27A9E0]
+          bg-white text-neutral-1000`
           : item.done
-          ? `border-secondary-500 bg-secondary-300 text-neutral-800  `
-          : `border-neutral-100 bg-neutral-300 text-neutral-900`
+          ? `border border-green-100  bg-green-50 `
+          : `bg-[rgba(0, 0, 0, 0.03)] border-b-[rgba(0, 0, 0, 0.03)] border-l-4 border-b-2 border-white border-l-transparent text-neutral-900`
```

Comment:
There doesn't appear to be any need for an interpolation string on these two, please use `"` instead.

Reply:
done